warehouse/query-core/src/main/java/datawave/query/attributes/HitTermType.java

```diff
@@ -0,0 +1,11 @@
+package datawave.query.attributes;
+
+import datawave.data.type.StringType;
+
+import java.util.function.Predicate;
+
+public class HitTermType extends StringType {
```

Comment:
Let go ahead and move this to Type-utils now.  Releasing a new version of that and updating dependencies in the datawave repo is not hard to do.

Reply:
Shall I open a PR in datawave-type-utils?